src/__test__/commentCount.test.js

```diff
@@ -0,0 +1,32 @@
+import { testComments } from '../modules/commentCount.js';
```

Comment:
- [ ] Kindly update the test for the count of all comments based on the new implementation and please make sure that your test includes all edge cases like if there are 0 items on the page.